README.md

```diff
@@ -8,10 +8,10 @@ about the project. The project is currently being developed using Hugo based on
 The project follows the general structure of a `hugo` project, and to work with the codebase, you will need to [install hugo v0.93.3](https://gohugo.io/getting-started/installing/) (note if downloading the binary directly, you need to download the extended version to get live SCSS reload). Once installed, you should be able to run `hugo server` to run a local version of the project. 
 
 ## Install steps
-wget https://github.com/gohugoio/hugo/releases/download/v0.93.3/hugo_0.93.3_Linux-64bit.deb
-sudo dpkg -i hugo_0.93.3_Linux-64bit.deb
-then make your directory, clone cdev-website then cd into it
-hugo server
+```wget https://github.com/gohugoio/hugo/releases/download/v0.93.3/hugo_0.93.3_Linux-64bit.deb
```

Comment:
Hi Ian, I am sorry for any confusion. I meant for Line 11, 12 & 14, wrap each line in 3 backticks to denote a code block. I have included an image of an example below.   
11
[![image](https://user-images.githubusercontent.com/66088725/178757184-81cfe340-bdef-487b-b87d-df3aadd12525.png)](https://raw.githubusercontent.com/cdev-framework/assemblit/main/CONTRIBUTING.md?token=GHSAT0AAAAAABH4LER5TNJWQLGUO3FLYRFOYWO2VWA)